dao/pom.xml

```diff
@@ -79,6 +79,13 @@
                         </goals>
                     </execution>
                 </executions>
+                <configuration>
+                    <args>
+                        <arg>-Xfatal-warnings</arg>
+                        <arg>-unchecked</arg>
+                        <arg>-deprecation</arg>
```

Comment:
Can we get a quick link or explanation?

Reply:
https://docs.scala-lang.org/overviews/compiler-options/index.html

`-Xfatal-warnings` makes the compilation fail on warnings
`-unchecked` - adds location and describes in detail warnings where generated code depends on assumptions
`-deprecation` - adds location and describes in detail the deprecation warning

Actually should have included `-feature` more than `-unchecked` as it is the one we had experienced together with `-deprecation`. I will add it.
For the record, including the switches does not change if the warning is generated, but a difference between:
_There were 2 deprecation warnings_
vs.
_There's a deprecation warning in file XYZ using class OPQ on line 123 and 301._